site/public/js/pending-gradeables.js

```diff
@@ -0,0 +1,74 @@
+function sortTableByColumn(sortKey) {
+    const currentSort = Cookies.get('pending_gradeable_table_key');
+    const currentDirection = Cookies.get('pending_gradeable_table_direction') || 'ASC';
+
+    let newDirection;
+    if (currentSort === sortKey) {
+        newDirection = (currentDirection === 'ASC' ? 'DESC' : 'ASC');
+    }
+    else {
+        newDirection = 'ASC';
+    }
+
+    Cookies.set('pending_gradeable_table_key', sortKey, { path: '/superuser/gradeables' });
```

Comment:
Since the cookie name would only be used on the pending gradeables page, it might make sense to set path to `/` to mimic a lot of other cookies on our site